#!/usr/bin/python3
def raise_exception():
    result = 5 + "school"
    return result
